rules/windows/file/file_event/file_event_win_creation_deno.yml

```
@@ -0,0 +1,28 @@
+title: Suspicious Deno File Written from Remote Source
+id: 6c0ce3b6-85e2-49d4-9c3f-6e008ce9796e
+status: experimental
+description: |
+    Detects Deno writing a file from a direct HTTP(s) call and writing to the appdata folder or bringing it's own malicious DLL.
+    This behavior may indicate an attempt to execute remotely hosted, potentially malicious files through deno.
+references:
+    - https://taggart-tech.com/evildeno/
+author: Josh Nickels, Michael Taggart
+date: 2025-05-06
+logsource:
+    category: file_event
+    product: windows
+ tags:
+    - attack.execution
+    - attack.t1204
+    - attack.t1059.007
+    - attack.command-and-control
+    - attack.t1105
```

```suggestion
 tags:
    - attack.execution
    - attack.t1204
    - attack.t1059.007
    - attack.command-and-control
    - attack.t1105
logsource:
    category: file_event
    product: windows
```

Sorry, the tags should be before logsource